futaba/cogs/moderation/core.py

```diff
@@ -201,6 +211,17 @@ async def jail(self, ctx, member: MemberConv, *, reason: str = None):
 
         await self.perform_jail(ctx, member, 0, reason)
 
+    async def on_message(self, message):
+        """
+        Bot Event.
+        :param message: Messages.
+        :return: Nothing.
+        """
+        if len(message.mentions) > 5:
```

Comment:
On another though I probably should ensure that this line does not count the same mention more than once like for example if they did ``@sameperson @sameperson @sameperson`` more than once.